sql: TestTxnUserRestart failed

sql.TestTxnUserRestart [failed](https://tanzanite.cluster.engflow.com/invocations/default/1177f4db-c123-4d14-9156-1304a0afc3b4?testReportRun=2&testReportShard=5&testReportAttempt=1#targets-Ly9wa2cvc3FsOnNxbF90ZXN0) on master @ [e215cf435851cb9490fe4494fe73d02041c4d2c0](https://github.com/cockroachdb/cockroach/commits/e215cf435851cb9490fe4494fe73d02041c4d2c0):

Fatal error:

```
panic: test timed out after 39m57s
running tests:
	TestTxnUserRestart (4s)
	TestTxnUserRestart/err=RETRY_REASON_UNKNOWN,stgy=1 (1s)

```
Stack: 

```
goroutine 246826361 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x385
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestTxnUserRestart
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestTxnUserRestart2605058918
    test_log_scope.go:81: use -show-logs to present logs inline
=== RUN   TestTxnUserRestart/err=RETRY_REASON_UNKNOWN,stgy=0
=== RUN   TestTxnUserRestart/err=RETRY_REASON_UNKNOWN,stgy=1
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=5</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTxnUserRestart.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-38115

--- a/pkg/sql/mvcc_statistics_update_job_test.go
+++ b/pkg/sql/mvcc_statistics_update_job_test.go
@@ -49,7 +49,7 @@ func TestTenantGlobalAggregatedLivebytes(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
-	skip.UnderRace(t, "test is too slow to run under race")
+	skip.UnderDuress(t, "too slow")
 
 	ctx := context.Background()
 	jobID := jobs.MVCCStatisticsJobID

